src/components/interface/Tabs/Tabs.js

```diff
@@ -0,0 +1,43 @@
+import classnames from 'classnames';
+import {
+  cloneElement, useState, Children,
+} from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import PropTypes from 'prop-types';
+import { CHILDREN_TYPE, CLASS_NAME_TYPE, SCHEME_TYPE } from '../../../utils/variables';
+
+const Tabs = ({
+  className, children, defaultActiveTab, scheme,
+}) => {
+  const [activeTab, setActiveTab] = useState(() => defaultActiveTab);
```

Comment:
pourquoi des fonction ici ?

Reply:
Ça permet de limiter limiter l'assignment de la variable à l'init. https://blog.logrocket.com/a-guide-to-usestate-in-react-ecb9952e406c/